packages/core/src/components/Onboarding/OnboardingContainer.tsx

```diff
@@ -39,33 +39,41 @@ export function OnboardingContainer({
           height: '100%',
           width: '100%',
           zIndex: -1,
-          background:
-            'linear-gradient(145deg, transparent, lightgray 70%, lightgray)',
+        }}
+        background={{
+          base: 'unset',
+          md: 'linear-gradient(145deg, transparent, lightgray 70%, lightgray)',
         }}
       />
 
       <Box
         css={{
           backgroundColor: 'var(--background-color-primary)',
-          padding: '64px 128px',
           borderRadius: 8,
-          height: '50%',
         }}
         shadow="2xl"
         width={{
           base: '100%',
           md: '80%',
           lg: '75%',
         }}
+        height={{
+          base: '100%',
+          md: '50%',
+        }}
         minHeight={'600px'}
-        minWidth={'700px'}
+        minWidth={[null, '700px']}
```

Comment:
so what's the idea here, `sm: is null` and then it's 700px for md and above? 

A nit but, I think I prefer the object syntax. Just makes it more readable: https://chakra-ui.com/docs/styled-system/responsive-styles#the-object-syntax

Just noting it for moving forward.

Reply:
Done.